test-network/network.sh

```diff
@@ -479,6 +479,7 @@ else
   exit 1
 fi
 
```

Comment:
Does COMPOSE_PROJECT_NAME=fabric need to be in both places? Could it be just in the env file?

Wouldn't this update change the name of the docker composer network? The docs refer to the `net_test` docker compose network in quite a few places?

https://hyperledger-fabric.readthedocs.io/en/latest/search.html?q=net_test&check_keywords=yes&area=default

Reply:
let's see this CI log https://dev.azure.com/Hyperledger/Fabric-Samples/_build/results?buildId=29892&view=logs&j=98b5b0e0-fc73-50d8-cdbe-b7a1b46de137&t=e8322841-a67d-5ec3-ae89-37f2233483b3&l=513 with our code
https://github.com/hyperledger/fabric-samples/blob/master/test-network/.env#L1

ref here:
https://dev.azure.com/Hyperledger/TWGC/_build/results?buildId=29851&view=logs&j=079c46b8-677c-5016-d763-eb78926de79e&t=42fc6774-e221-5249-c003-7cb87b0a6fd3&l=886
the on release-1.4 branch, `net_byfn` is the network name and works.

wait a min....